File: _stats_build/statistics/wr_ao3r.rb
require_relative "abstract/ao_rounds"

class WrAo3r < AoRounds
  def initialize
    @title = "World record Ao3R (Average of 3 Rounds) history"
    @note = "Ao3R: average of the 3 round averages (R1 + R2 + Final) in a competition."
    @table_header = { "Result" => :right, "Improvement" => :right, "Days" => :right, "Person" => :left, "Date" => :left, "Competition" => :left, "Details" => :left }
  end

  def round_count
    3
  end
end

File: core/packages/stats-build/src/statistics/wr_ao3r.ts
// NOTE: Ao3R——一场比赛恰好 3 轮时各轮 average 的均值
import { AoRounds } from '../core/ao_rounds.js';

export class WrAo3r extends AoRounds {
  constructor() {
    super();
    this.title = 'Ao3R';
    this.titleZh = 'Ao3R';
    this.note = 'Average of 3 Rounds: mean of averages when a person competed in exactly 3 rounds at a competition.';
    this.noteZh = '三轮平均：一场比赛中恰好参加 3 轮时各轮平均的均值。';
  }
  roundCount() { return 3; }
}
